backend/stakepoold/server.go

```diff
@@ -253,18 +254,16 @@ func evaluateStakePoolTicket(ctx *appContext, tx *wire.MsgTx, blockHeight int32)
 }
 
 // MsgTxFromHex returns a wire.MsgTx struct built from the transaction hex string
-func MsgTxFromHex(txhex string) *wire.MsgTx {
+func MsgTxFromHex(txhex string) (*wire.MsgTx, error) {
 	txBytes, err := hex.DecodeString(txhex)
 	if err != nil {
-		log.Warnf("DecodeString failed for %v: %v", txhex, err)
-		return nil
+		return nil, err
 	}
 	msgTx := wire.NewMsgTx()
 	if err = msgTx.Deserialize(bytes.NewReader(txBytes)); err != nil {
-		log.Warnf("Deserialize failed for %v: %v", txBytes, err)
-		return nil
+		return nil, err
 	}
-	return msgTx
+	return msgTx, nil
```

Comment:
I meant to do the following in `dcrdata/txhelpers.MsgTxFromHex`, but it's not important:

```go
msgTx := wire.NewMsgTx()
err := msgTx.Deserialize(hex.NewDecoder(strings.NewReader(txhex)))
return msgTx, err
```